src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/AS_RankSumTest.java

```diff
@@ -112,13 +105,10 @@ protected String makeRawAnnotationString(final List<Allele> vcAlleles, final Map
 
     // Generates as CompressedDataList over integer values over each read
     @SuppressWarnings({"unchecked", "rawtypes"})//FIXME generics here blow up
-    public void calculateRawData(VariantContext vc, final ReadLikelihoods<Allele> likelihoods, ReducibleAnnotationData myData) {
+    private void calculateRawData(VariantContext vc, final ReadLikelihoods<Allele> likelihoods, ReducibleAnnotationData myData) {
         if( vc.getGenotypes().getSampleNames().size() != 1) {
             throw new IllegalStateException("Calculating raw data for allele-specific rank sums requires variant context input with exactly one sample, as in a gVCF.");
         }
-        if(likelihoods == null) {
```

Comment:
I would leave this in case someone in the future tries to call this allele-specific annotation without likelihoods, but I have conservative leanings in my programming.

Reply:
Correct, I'm not sure why I deleted this but i've added it back